_episodes/07-testing.md

```diff
@@ -776,7 +787,7 @@ For some of our functions like `calculate_distance` or `calculate_angle`, we hav
 {: .callout}
 
 
-This is called the `pytest.mark.parametrize` decorator. The syntax for this decorator is
+The syntax for the `pytest.mark.parameterize` decorator is:
```

Comment:
It actually is `parametrize` instead of `parameterize`. Trips me up all the time - https://docs.pytest.org/en/latest/parametrize.html

Reply:
Argh... Thanks for spotting this, I make this mistake all the time..